core/store/migrations/migration1581240419/migrate.go

```diff
@@ -0,0 +1,29 @@
+package migration1581240419
+
+import (
+	"chainlink/core/store/dbutil"
+	"github.com/jinzhu/gorm"
+)
+
+// Migrate moves job_runs.overrides to run_requets.request_params
+func Migrate(tx *gorm.DB) error {
+	err := tx.Exec(`ALTER TABLE run_requests ADD request_params text NOT NULL DEFAULT '{}'`).Error
+	if err != nil {
+		return err
+	}
+	err = tx.Exec(`
+		UPDATE run_requests
+		SET request_params = (
+			SELECT overrides
+			FROM job_runs
+			WHERE run_request_id = run_requests.id
+		)
+	`).Error
+	if err != nil {
+		return err
+	}
+	if dbutil.IsPostgres(tx) {
+		return tx.Exec(`ALTER TABLE job_runs DROP COLUMN IF EXISTS overrides`).Error
```

Comment:
I believe dropping a column goes against our zero downtime policy

Reply:
I don't really understand why it would...

In fact I don't really understand the zero downtime policy... since it's impossible to deploy the app without downtime anyway, because it _must_ be restarted to run migrations.

Reply:
Its what I've understood from our earlier discussions.  I'd suggest just leaving this line out for now.

There's a [story](https://www.pivotaltracker.com/story/show/171085685) for documenting our migration practices.  Maybe we can bring this up then?  It would be a good place to focus the discussion around.

Reply:
Think of it more as a minimal downtime policy ;) I'd just want to test this on a production clone to make sure it doesn't take too long (< 1 minute). Otherwise it will impact overall uptime.

Reply:
I have tested it on a prod clone. See numbers in this comment: https://github.com/smartcontractkit/chainlink/pull/2267#issuecomment-584669224